src/components/Navmenu.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { uniq, includes } from 'lodash';
+import CloseSVG from 'svg/Close';
+
+class Navmenu extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      nav: []
+    };
+  }
+
+  menuClassName = () => {
+    const baseClassName = `coa-Navmenu`;
+    return this.props.isOpen ? `${baseClassName}--open` : baseClassName;
```

Comment:
For css convention — I was going with BEM class stacking (http://getbem.com/introduction/). So className=“base_class base_class—modifier”.  NOT just className=“base_class—modifier” 
Honestly, I haven’t spent a lot of time thinking/reading about what’s better here, just leaning on the folks who came up with BEM stuff to have done that heavy lifting.

Reply:
yeah, I don't really have a strong opinion here. I updated for this case, but unless there is a clear benefit, i'd leave it to chef's choice.